### PR TITLE
Maintenance/update dependencies

### PR DESCRIPTION
**Pull request by azriel91**
## Description

Updated dependencies:

```
* itertools:   0.7    -> 0.8
* parking-lot: 0.6    -> 0.9
* image:       0.21.0 -> 0.22.2
* getset:      0.0.7  -> 0.0.8
* sdl2:        0.31.0 -> 0.32.2
* shred:       0.7    -> 0.9
* criterion:   0.2.11 -> 0.3
* glyph_brush: 0.5.0  -> 0.6.0
```

## PR Checklist

By placing an x in the boxes I certify that I have:

- **n/a** Updated the content of the book if this PR would make the book outdated.
- **n/a** Added a changelog entry if this will impact users, or modified more than 5 lines of Rust that wasn't a doc comment.
- **n/a** Added unit tests for new code added in this PR.
- [x] Acknowledged that by making this pull request I release this code under an MIT/Apache 2.0 dual licensing scheme.

If this modified or created any rs files:

- **n/a** Ran `cargo +stable fmt --all`
- **n/a** Ran `cargo clippy --all --features "empty"`
- [x] Ran `cargo test --all --features "empty"`

### CODE REVIEW

File: amethyst_rendy/Cargo.toml [hunk 1]
```diff
@@ -44,6 +44,7 @@ winit = "0.19"
 approx = "0.3"
 
 [features]
+default = ["vulkan"]
```
Comment (distransient) [inline amethyst_rendy/Cargo.toml]: doesn't this require non-vulkan users to use default-features = false?

Comment (azriel91) [reply amethyst_rendy/Cargo.toml]: oops, used that for my local testing

**Comment by azriel91** [comment]
Linux:

```
final link failed: No space left on device
```

Windows:

```
No space left on device
```

Cleaning disk space now.

**Comment by azriel91** [comment]
bors retry

**Pull request merged**